bin/bib_newyear.pl

```diff
@@ -70,8 +70,7 @@ END
           ' INNER JOIN reasons rs ON r.reason=rs.id'.
           ' WHERE CONCAT(a.name,"/",rs.name)'.
           ' IN ("ic-world/con","ic/cdpp","ic/crms","ic/ipma","ic/ren","op/ipma",
```

Comment:
I would prefer to have this one item per line, like:

```sql
IN (
  "ic-world/con",
  "ic/cdpp",
  ...
)
```
... so that any future diffs will be easier to read.